fs/index.ts

```diff
@@ -4,6 +4,16 @@ export interface IStudent {
   lastName: string;
   password: string;
   email: string;
+  discordId?: string;
+  location?: string;
+  personalWebsite?: string;
+  hobbies?: string;
+  githubName?: string;
+  about?: string;
+  handle?: string;
```

Comment:
what is handle?

Reply:
username. I saw it like that in the template code of settings so I just changed username to handle

Reply:
nah, use `username` instead.

Reply:
alright